src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -256,4 +260,14 @@ public ResponseEntity<List<String>> getUserInfo(
         return ResponseEntity.ok(userInformationService.getUserCaseRoles(caseId, authorization));
     }
 
+    @PostMapping(path = "/{caseId}/courtDecision")
+    @Operation(summary = "Get decision on proposed claimant repayment")
+    public RepaymentDecisionType getDecisionOnClaimantProposedRepayment(
+        @PathVariable("caseId") Long caseId,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestBody ClaimantProposedPlan claimantProposedPlan) {
+        var caseDetails = coreCaseDataService.getCase(caseId, authorisation);
+        return repaymentPlanDecisionService.getCalculatedDecision(caseDetails, claimantProposedPlan);
```

Comment:
we could wrap on ResponseEntity
return ResponseEntity.ok(decision);

Reply:
I dont think its a good approach Leo. it will return ok anyway if it is successful. we are returning a dto (json) here. have a look at other post methods

Reply:
What i mean is we are implicitly returning a responsebody here (because it is a rest controller). so the response will be serialized into the body of the HTTP response. hence we dont need a response entity here